player.cpp

```diff
@@ -133,8 +133,9 @@ void Player::onEvent(const GameEvent& event) {
         }
       },
       [&](const OtherEffect& info) {
-        if (creature->canSee(info.position))
-          getView()->animation(info.position.getCoord(), info.effect, info.targetOffset);
+        // Effects were spawned twice: once from Player, once from PlayerControl
```

Comment:
This won't cut it, in adventurer mode there is no PlayerControl. Please uncomment, I'll fix it in another commit.